### PR TITLE
SQLhomework

### PR DESCRIPTION
**Pull request by Yanetta**

### CODE REVIEW

File: src/main/resources/SQLhomework.sql [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+SELECT
+    product,
+    COUNT(product)
+FROM
+    orders
+GROUP BY
+    product
+HAVING
+    COUNT(product) > 2;
+
```
Comment (un-install) [inline src/main/resources/SQLhomework.sql:209]: more then two times or three

**Review by un-install — CHANGES_REQUESTED** [review]
update/select/delete queries should have more various logic.
there should be few more queries (8 - 9)

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Don't see all request

**Review by un-install — APPROVED** [review]

**Review by SerhiiPasko — APPROVED** [review]
Good

**Pull request merged**